.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,14 @@
-Fix # .
+Fix issue # .
+
+Short description of the fix:
+
+- [ ] I ran all unit tests locally
+- [ ] CHANGELOG.md updated or do not need to be updated
+  - If not tell why: 
 
 
 For significant contributions please make sure you have completed the following items:
 
-- [ ] Design discussion issue #
 - [ ] Changes in public surface reviewed
-- [ ] CHANGELOG.md updated
-- [ ] Ran atleast all unit tests locally
-- [ ] Ran functional tests locally if making major changes.
-- [ ] The PR will trigger build, unit test, and functional tests automatically. If you are non-Microsoft employee, please
-mention any committers to help trigger build.
+- [ ] I ran [functional tests](https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/develop/CONTRIBUTING.md#functional-tests) locally.
+- [ ] The PR will trigger build, unit test, and functional tests automatically. If you are non-Microsoft employee, please mention any committers to help trigger build.
```

Comment:
If you are non-Microsoft employee or if you are submitting PR from a forked repository

Reply:
fixed